doc/api/http.md

```diff
@@ -50,6 +50,10 @@ list like the following:
 ## Class: http.Agent
 <!-- YAML
 added: v0.3.4
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/commit/186035243fad247e3955f
+    description: The maximum size of HTTP/1 headers is now 8KB.
```

Comment:
This doesn't seem like the right place. It's a property of the underlying HTTP APIs, not of the agent, isn't it?

Reply:
Right. Are you referring to move it under `Class: http.ClientRequest` ?

Reply:
actually, yes because that will also be consistent with the server change too.

Reply:
yes, agreed, that would be a better place.